child-logger.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var defaultLevels = require('./default-levels.js');
+var makeLogMethod = require('./log-method');
+var errors = require('./errors');
+
+module.exports = ChildLogger;
+
+function ChildLogger(config) {
+    this.mainLogger = config.mainLogger;
+    this.path = config.path;
```

Comment:
how do we avoid namespace conflicts? e.g. is it likely two child loggers will get the same path and do we care? I assume that two separate child loggers with the same path are not a problem